code/game/objects/structures/signs.dm

```diff
@@ -1235,6 +1235,10 @@
 	name = "CHAPEL"
 	icon_state = "chapel"
 
+/obj/structure/sign/department/janitor
+	name = "JANITOR"
+	icon_state = "janitor"
+

```

Comment:
```suggestion
//YW Addition start
/obj/structure/sign/department/janitor
	name = "JANITOR"
	icon_state = "janitor"
	icon = 'icons/obj/decals_yw.dmi'
//YW Addition End

```
You will have to make icons/obj/decals_yw.dmi and put your icon in there and revert the change to icons/obj/decals_vr.dmi we don't want to change stock vore dmi files as they will get reverted every time they make a change to them.